### PR TITLE
Generate ConfigMap manifest

### PR DESCRIPTION
**Pull request by chamer81**
This is a simple implementation of adding a configmap with one demo environment variable.

### CODE REVIEW

**Comment by bittner** [comment]
Just for transparency's sake, this would be the resulting directory structure:
```console
$ tree -a \{\{cookiecutter.project_slug\}\}/_/deployment/python/deployment/application/
{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/
├── base
│   ├── deployment.yaml
│   ├── kustomization.yaml
│   ├── rolebinding.yaml
│   ├── route-crd.yaml
│   ├── route.yaml
│   ├── secret.yaml
│   └── service.yaml
├── Dockerfile
├── entrypoint.sh
├── overlays
│   ├── development
│   │   ├── .env
│   │   └── kustomization.yaml
│   ├── integration
│   │   ├── .env
│   │   └── kustomization.yaml
│   └── production
│       ├── deployment.yaml
│       ├── .env
│       └── kustomization.yaml
└── uwsgi.ini
```

**Comment by bittner** [comment]
BTW, in our demo the `DJANGO_DEBUG` value would be used in the Django [application settings](https://gitlab.com/-/ide/project/appuio/example-django/blob/master/-/application/settings.py).

**Comment by srueg** [comment]
I don't really like the `.env` filenames. To me it creates the impression of only locally used files. Also they're hidden by default (because of the leading `.`).
I'd suggest to rename them to something like `$appname.env` maybe?

File: {{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/kustomization.yaml [hunk 1]
```diff
@@ -5,14 +5,18 @@ commonLabels:
...
-- deployment.yaml
-- route.yaml
-- secret.yaml
-- service.yaml
-- rolebinding.yaml
+  - deployment.yaml
+  - route.yaml
+  - secret.yaml
+  - service.yaml
+  - rolebinding.yaml
```
Comment (bittner) [inline {{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/kustomization.yaml]: The change of the indentation is probably unnecessary. I believe, we try to keep the indentation to the minimum needed in YAML.

**Pull request merged**